content/upbound-marketplace/providers.md

```diff
@@ -2,57 +2,182 @@
+the Provider.
+
+```yaml {label="air"}
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: upbound-provider-aws-s3
+spec:
+  package: xpkg.upbound.io/upbound/provider-aws-s3:v0.37.0
+  skipDependencyResolution: true
+```
+
+When installing multiple providers, apply `skipDependencyResolution` to every
+Provider.
 
 ```yaml
 apiVersion: pkg.crossplane.io/v1
 kind: Provider
 metadata:
-    name: provider-aws
+  name: upbound-provider-aws-s3
 spec:
-  package: xpkg.upbound.io/upbound/provider-aws:v0.17.0
-  packagePullSecrets:
-    - name: package-pull-secret
+  package: xpkg.upbound.io/upbound/provider-aws-s3:v0.37.0
+  skipDependencyResolution: true
+---
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: upbound-provider-aws-ec2
+spec:
+  package: xpkg.upbound.io/upbound/provider-aws-ec2:v0.37.0
+  skipDependencyResolution: true
+```
+
+{{< /expand >}}
+
+View the installed providers with `kubectl get providers`
+
+```shell {copy-lines="1",label="getproviders"}
+kubectl get providers
+NAME                          INSTALLED   HEALTHY   PACKAGE                                               AGE
+upbound-provider-aws-ec2      True        True      xpkg.upbound.io/upbound/provider-aws-ec2:v0.37.0      25m
+upbound-provider-aws-s3       True        True      xpkg.upbound.io/upbound/provider-aws-s3:v0.37.0       25m
+upbound-provider-family-aws   True        True      xpkg.upbound.io/upbound/provider-family-aws:v0.37.0   24m
+```
+
+The first provider installed of a family also installs an extra
+{{<hover label="getproviders" line="5">}}provider-family{{</hover>}} Provider.
+The `provider-family` provider manages the ProviderConfig
+for all service providers in the same family. 
+
+### Using ControllerConfigs
+
+The ControllerConfig applies settings to a Provider Pod. With family providers
+each service is an unique Pod running in the cluster. 
+
+When using ControllerConfigs with family providers, each provider needs
+a ControllerConfig applied.
+
+For example, both service providers reference the same 
+{{<hover label="cc" line="8">}}controllerConfigRef.name{{</hover >}}.
+
+```yaml
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: upbound-provider-aws-s3
+spec:
+  package: xpkg.upbound.io/upbound/provider-aws-s3:v0.37.0
+  controllerConfigRef:
+    name: my-controllerconfig
+---
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: upbound-provider-aws-ec2
+spec:
+  package: xpkg.upbound.io/upbound/provider-aws-ec2:v0.37.0
+  controllerConfigRef:
+    name: my-controllerconfig
+---
+apiVersion: pkg.crossplane.io/v1alpha1
+kind: ControllerConfig
+metadata:
+  name: my-controllerconfig
 ```
-{{< /tab >}}
 
-{{< /tabs >}}
+## Monolithic official providers
+
+Upbound delivered the official providers as a single monolithic provider
+beginning in October, 2022. Starting June 13, 2023 Upbound split the official 
+providers into smaller family providers. 
```

Comment:
```suggestion
Upbound delivered the AWS, GCP and Azure official providers as a single 
monolithic provider beginning in October, 2022. Starting June 13, 2023 
Upbound split these official providers into provider families consisting 
out of many smaller providers. 
```
Since we have official providers that are not being split (provider-azuread and provider-terraform) I feel it's worth qualifying which providers are being split.